lib/puppet/pops/serialization/extension.rb

```diff
@@ -26,8 +27,8 @@ module Extension
   TIMESPAN = 0x34
   VERSION = 0x35
   VERSION_RANGE = 0x36
-  SENSITIVE = 0x37
-  BINARY = 0x38
+  BINARY = 0x37
+  BASE64 = 0x38
```

Comment:
It is, and a binary capable serializer (e.g. msgpack) will use BINARY whereas a non-binary serializer (e.g. json) will use BASE64 when writing to a stream. But consider what happens if one version of a serializer isn't capable whereas of writing binary content but another version is. The latter would then need to differentiate between the two when reading. Using two separate extension points (both for the Binary) makes this possible.